### DISCUSSION TITLE
[Feature Request] Extend reference popups to include other referene elements

### DISCUSSION THREAD

**Discussion by agerlach** [quarto-dev/quarto-cli#3712] [Feature Requests]
Reference Popups for html output are fantastic. It really improve the reading experience over a static document. I would love to see them extended to other reference elements as well, e.g., equations, figures, tables, [theorems and proofs divs](https://quarto.org/docs/authoring/cross-references.html#theorems-and-proofs).

**Comment by mcanouil** [comment]
FYI, related to #2427 in some extent.

**Reply by agerlach** [reply]
Awesome. Thanks for heads up
